cmds/core/mv/mv_test.go

```diff
@@ -26,44 +25,41 @@ var hiFileContent = []byte("hi")
 
 var old = makeIt{
 	n: "old.txt",
-	m: 0777,
+	m: 0o777,
 	c: []byte("old"),
 }
 
 var new = makeIt{
 	n: "new.txt",
-	m: 0777,
+	m: 0o777,
 	c: []byte("new"),
 }
 
 var tests = []makeIt{
 	{
 		n: "hi1.txt",
-		m: 0666,
+		m: 0o666,
 		c: hiFileContent,
 	},
 	{
 		n: "hi2.txt",
-		m: 0777,
+		m: 0o777,
 		c: hiFileContent,
 	},
 	old,
 	new,
 }
 
-func setup() (string, error) {
-	d, err := ioutil.TempDir(os.TempDir(), "hi.dir")
-	if err != nil {
-		return "", err
-	}
+func setup(t *testing.T) (string, error) {
+	d := t.TempDir()
 
 	tmpdir := filepath.Join(d, "hi.sub.dir")
-	if err := os.Mkdir(tmpdir, 0777); err != nil {
+	if err := os.Mkdir(tmpdir, 0o777); err != nil {
 		return "", err
 	}
 
 	for _, t := range tests {
```

Comment:
Rename this to tt because the way it is shadowing the other variable named t is a bit confusing.